contrib/inventory/ovirt.py

```diff
@@ -172,9 +172,9 @@ def get_ovirt_driver():
 
         # If the appropriate environment variables are set, they override
         # other configuration; process those into our args and kwargs.
-        kwargs['url'] = os.environ.get('OVIRT_URL')
-        kwargs['username'] = os.environ.get('OVIRT_EMAIL')
-        kwargs['password'] = os.environ.get('OVIRT_PASS')
+	kwargs['url'] = environ.get('OVIRT_URL', kwargs['url'])
+	kwargs['username'] = environ.get('OVIRT_USERNAME', kwargs['username'])
+	kwargs['password'] = environ.get('OVIRT_PASSWORD', kwargs['password'])
```

Comment:
Isn't it missing a whitespace ?

Are you sure environ is defined here ? I don't see it.

Perhaps this would work too, if you do want OVIRT_URL= to override kwargs['url']:

kwargs['url'] = environ.get('OVIRT_URL', kwargs['url'])
Example:

$ A= python -c 'import os; print os.environ.get("A", "X")'

(we get an empty string)

If you do not want OVIRT_URL= to override kwargs['url'], then you can use or:

kwargs['url'] = os.environ.get('OVIRT_URL') or kwargs['url']
Example:

$ A= python -c 'import os; print os.environ.get("A") or "X"'
X
(we get the value that's at the right of or)

Hope this helps